### ISSUE TITLE
tflearn import issue

### ISSUE THREAD

**Issue by krinali1901** [tflearn/tflearn#1172]
Hello,

I am trying to run a code using tflearn. But even if i just try to import tflearn then it gives the below error
WARNING:tensorflow:From /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/tensorflow/python/compat/v2_compat.py:107: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term

mportError: cannot import name 'is_sequence' from 'tensorflow.python.util.nest' (/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/tensorflow/python/util/nest.py)

What can be done to resolve this?

**Comment by rela-v** [comment]
Hi! Thanks for your question.

### tl;dr: everywhere you say "is_sequence" in your code, replace it with "is_sequence_or_composite".

### Your problem
- If you try to do the following:
`from tensorflow.python.util.nest import is_sequence`
- The following error results:
`ImportError: cannot import name 'is_sequence' from 'tensorflow.python.util.nest'`
`(/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/tensorflow/python/util/nest.py)`

### Explanation of the Problem
If this is indeed the problem, then I encountered the same issue several days ago and was able to resolve it. The reason you are experiencing the error is that, as the error message says, you 
`cannot import name 'is_sequence' from 'tensorflow.python.util.nest'`
which means that there was some issue in the import process. Indeed, cursory research reveals that there is no longer any name "is_sequence" in the nest.py module (per https://github.com/tensorflow/tensorflow/commit/f564400be34ed5c82f448699a2d91e1135d76f97).

### Solution
If you visit the provided link above, there is a new function which will serve your purpose well named "is_sequence_or_composite()", which according to the source code does the same exact thing as "is_nested_or_composite()" (use whichever one floats your boat!).
Thus, replacing the "is_sequence" function with the "is_sequence_or_composite" function will resolve the error.
For more information on what a "nested structure" is, which is what this function actually checks for, see the following link:
https://www.tensorflow.org/api_docs/python/tf/nest

Thank you so much again for asking. I hope this works for you, and feel free to ask for any help you may need along the way!

-Rela

**Comment by MihaMarkic** [comment]
@rela-v It happens if one doesn't use is_sequence explictly, simple `import tflearn` is enough to trigger the error. So, it is not possible to correct the error which is most probably somewhere in tflearn code.

**Comment by MihaMarkic** [comment]
And as expected, is_sequence is imported in [tflearn/layers/recurrent.py](https://github.com/tflearn/tflearn/blob/db5176773299b67a2a75c5889fb2aba7fd0fea8a/tflearn/layers/recurrent.py#L17) file.

**Comment by MihaMarkic** [comment]
Created a [PR](https://github.com/tflearn/tflearn/pull/1173)

**Comment by awsumbill** [comment]
In the meantime you can `pip install tensorflow<2.13` to get around this untill it's fixed
